code/game/objects/structures/crates_lockers/crates.dm

```diff
@@ -45,6 +45,13 @@
 	if(manifest)
 		tear_manifest(user)
 
+/obj/structure/closet/crate/tool_interact(obj/item/W, mob/user)
+	if(istype(W, /obj/item/wirecutters))
+		if (manifest)
+			tear_manifest(user)
+			return TRUE
+	return ..()
+
```

Comment:
Any reason not to use `wirecutter_act` and thus skipping the tipecheck?

Reply:
I tried it and it didn't work, so yes